### PR TITLE
Update exit-from-dialog.js

### PR DESCRIPTION
**Pull request by samanthaeschaffer**
Add exitHere and exitHereNow tags to allow for dialog-exits that don't specify coordinates.

### CODE REVIEW

File: dist/exit-from-dialog.js [hunk 1]
```diff
@@ -404,6 +412,43 @@ function _getExitParams(parameters) {
...
+function _getExitHereParams(parameters) {
+	var params = parameters[0].split(',');
+	var roomName = params[0];
+	var transition_effect = params[1];
+	var room = getRoom(roomName).id
+	
+	var x = player().x;
+	var y = player().y;
+
+	if (!roomName || x === undefined || y === undefined) {
```
Comment (seleb) [inline dist/exit-from-dialog.js]: since this is using the player position, the checks on position are unnecessary:

```suggestion
	if (!roomName) {
```

Comment (samanthaeschaffer) [reply dist/exit-from-dialog.js]: I'm pretty sure the position will still be undefined if you tried to use _exitHere_ in the title dialog, but I have to double check.

**Comment by samanthaeschaffer** [comment]
Ah, yup, my bad. I'll move that accordingly.~

The relative coordinates is a really interesting idea! If it's alright, I think I'll keep this PR as is, but I might submit that change as a future PR (if you don't beat me to it, haha).

File: dist/exit-from-dialog.js (hunk 1 above)
Comment (samanthaeschaffer) [reply dist/exit-from-dialog.js]: Checked again and it worked fine, so removed the position check in the latest commit.~

**Pull request merged**